tests/puppeteer/pages/BO/customerService/orderMessages/index.js

```diff
@@ -29,6 +29,10 @@ module.exports = class OrderMessages extends BOBasePage {
     this.dropdownToggleButton = `${this.actionsColumn} a.dropdown-toggle`;
     this.dropdownToggleMenu = `${this.actionsColumn} div.dropdown-menu`;
     this.deleteRowLink = `${this.dropdownToggleMenu} a[data-url*='/delete']`;
+    // Bulk Actions
+    this.selectAllRowsLabel = `${this.gridPanel} .md-checkbox label`;
+    this.bulkActionsToggleButton = `${this.gridPanel} button.js-bulk-actions-btn`;
+    this.bulkActionsDeleteButton = `${this.gridPanel} #order_message_grid_bulk_action_delete_selection`;
```

Comment:
```suggestion
    this.bulkActionsDeleteButton = '#order_message_grid_bulk_action_delete_selection';
```
it's an id, no need to have the gridPanel parent